packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -19,7 +19,7 @@ import { useLocation } from 'react-router';
 import { FetchStatus, SortType } from 'design/DataTable/types';
 import useAttempt from 'shared/hooks/useAttemptNext';
 import history from 'teleport/services/history';
-import { NodesResponse } from 'teleport/services/nodes';
+import { Node, NodesResponse } from 'teleport/services/nodes';
```

Comment:
I figure that we could  start using `import type` to be more explicit.

```suggestion
import type { Node, NodesResponse } from 'teleport/services/nodes';
```